### ISSUE TITLE
Display Parisian postal codes as arrondissement when displaying addresses on maps

### ISSUE THREAD

**Issue by jkotin** [Princeton-CDH/mep-django#509]
## notes for testing
Find a member with addresses and look at their map. Paris addresses should have arrondissement number as described below; no addresses should display postal codes.

----
Two connected issues: 1/ capitalization of "rue, "place," etc. is inconsistent; 2/ inclusion of postal codes in anachronistic—they were introduce in 1972.

For 1/ I think "rue" etc. should be lowercase throughout; and for 2/ ideally, we'd have 8e, for example, instead of 75008.

As [Wikipedia](https://en.wikipedia.org/wiki/Postal_codes_in_France) illustrates:

8 rue Chambiges
Paris 8e

**Comment by rlskoeser** [comment]
@jkotin capitalization inconsistencies should be fixable by updating the location entries in the database.

**Comment by clmahoney** [comment]
UPDATING NOW per RSK's request. 

Other things we know we want to do with data exports: 
- Identifying similar / repeat addresses
--> This will help us in Identifying relationships (if members lived together, stayed together, etc) 
-  Contribute to a new version of the map & geographic essay from the original MEP site

**Comment by rlskoeser** [comment]
@jkotin we were looking up french ordinal number digit abbreviations and wanted to confirm: is it _er_ for 1 (1er) and _e_ for everything else? Do you want the e/er superscripted, as it appears in the wikipedia article you referenced?

**Comment by jkotin** [comment]
Yes, correct.  And superscript would be ideal!

**Comment by rlskoeser** [comment]
@jkotin do you want postal codes displayed for non-Paris addresses, or should we just display nothing if there is no arrondissement?

**Comment by jkotin** [comment]
@clmahoney @i-davis Would you help advise on this? What did standard mailing addresses look like between 1920 and 1940 for non-Paris France, and for the U.S. and elsewhere?

**Comment by jkotin** [comment]
Zip codes in U.S. > 1963.

So I say no zip/postal codes ever for addresses.

**Comment by jkotin** [comment]
France: >1964.

Sorry for false alarm @clmahoney and @i-davis -- no need for additional research.

**Comment by jkotin** [comment]
This works great, and I'm happy to close. I did notice one thing: the leading on the superscript line is bigger than on the non-superscript line. I commented on a similar issue in Wagtail. See screenshot.  This isn't a blocker for me, but nice it could be fixed at some point.

<img width="798" alt="Screen Shot 2020-05-28 at 4 16 23 PM" src="https://user-images.githubusercontent.com/12616674/83189308-0a695000-a0ff-11ea-8105-0caae14d6937.png">
